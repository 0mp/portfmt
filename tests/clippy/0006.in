SUBPACKAGES=gdbm foo
COMMENT.gdbm=
COMMENT.foo=
