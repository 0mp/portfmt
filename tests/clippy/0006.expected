# Maintainer block
+COMMENT.foo
+COMMENT.gdbm

# Subpackages block
SUBPACKAGES

-COMMENT.gdbm
-COMMENT.foo
